package.json

```diff
@@ -49,47 +48,49 @@
   "homepage": "https://github.com/valor-software/ng2-table#readme",
   "dependencies": {},
   "peerDependencies": {
-    "@angular/common": "2.0.0-rc.6",
-    "@angular/compiler": "2.0.0-rc.6",
-    "@angular/core": "2.0.0-rc.6",
-    "@angular/forms": "2.0.0-rc.6"
+    "@angular/common": "2.0.1",
+    "@angular/compiler": "2.0.1",
+    "@angular/core": "2.0.1",
+    "@angular/forms": "2.0.1"
```

Comment:
Why not just use the caret or tilde syntax for Angular 2 peer deps? Updating to rely on an exact patch version will just create a need for new PRs and commits to update the peer deps every time Angular 2 gets a new patch.